components/html-block/html-block.js

```diff
@@ -161,16 +163,19 @@ class HtmlBlock extends RtlMixin(LitElement) {
 				text-align: left;
 			}
 			:host([inline]),
-			:host([inline]) div.d2l-html-block-rendered {
+			div.d2l-html-block-inline {
```

Comment:
Are these selectors overqualified? Could they just be:

```css
.d2l-html-block-inline {
   ...
}
.d2l-html-block-no-deferred-rendering {
   ...
}
.d2l-html-block-deferred-rendering {
   ...
}
```
Admittedly though, the `slot` element selector provides some hint/clarity.

Reply:
Yeah, they could be simplified. I always struggle at least a bit with how specific to get to avoid potential issues or pitfalls later. I'm content with whatever you would prefer.

Reply:
I'm good either way re the use of the `:host([no-deferred-rendering])` - there is a pro each way imo.

If keeping with the way it is, I'd reduce it to the minimum specificity that's required to avoid unnecessary selector checks (since `element.class-name` is in theory, two selector checks, ignoring possible optimizations the browser may or may not be doing). 🤷‍♂️

Reply:
Seems good. The slot has the opposite condition, which I could either accomplish with `:host(:not([no-deferred-rendering])) slot` or by keeping it as is (or I guess by using two separate selectors). Do you have a particular preference there? I think using the generated conditional class name means one selector and otherwise we need two, but... 🤷

Reply:
I don't feel strongly either way. I'm mostly focused on reducing the selectors to what's necessary, whichever way it's written. I do like relying on single source of truth (the host attribute) and avoiding conditional rendering which adds very slight complexity at the expense of perhaps an extra bit of CSS. I think the `:not()` could be avoided, but I really think it's splitting hairs we can't even measure.

Reply:
Yeah, fair enough.